### PR TITLE
Updated default value for timepicker to round up to next hour

### PR DESCRIPTION
**Pull request by blackbaud-GavinNicol**
Updated timepicker default value to round up to the next hour.

Addresses issue: #61

### CODE REVIEW

File: src/app/public/modules/timepicker/timepicker.component.ts [hunk 1]
```diff
@@ -48,6 +48,23 @@ export class SkyTimepickerComponent implements OnInit {
     this.setFormat(this.timeFormat);
   }
 
+  public get activeTime(): Date {
```
Comment (Blackbaud-ErikaMcVey) [inline src/app/public/modules/timepicker/timepicker.component.ts]: this looks good, could you just add a couple unit tests for the rounding logic?

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-contrib-update-timepicker-default-value](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-contrib-update-timepicker-default-value)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-contrib-update-timepicker-default-value](https://developer.blackbaud.com/skyux-datetime?_sp.spa=pr-contrib-update-timepicker-default-value)

**Comment by Blackbaud-AlexKingman** [comment]
Closing due to inactivity.

**Pull request closed without merging**